### ISSUE TITLE
encoding issue introduced in v2.5.1

### ISSUE THREAD

**Issue by tiejunhu** [maximdanilchenko/aiochclient#101]
When reading unicode string from clickhouse, the string in map is not correctly decoded.

With the following code:

```python
import asyncio

from aiochclient import ChClient
from aiohttp import ClientSession, ClientTimeout

"""
CREATE DATABASE test
USE test
CREATE TABLE test
(
    `title` String,
    `extra` Map(String, String)
)
ENGINE = EmbeddedRocksDB
PRIMARY KEY title
"""


async def get_client():
    timeout = ClientTimeout(total=5)
    session = ClientSession(timeout=timeout)
    client = ChClient(session, url="http://127.0.0.1:18123", database="test")
    return client

def assertEqual(str1, str2):
    if str1 != str2:
        raise Exception(f"{str1} != {str2}")

async def test_str(client):
    title = "测试"
    extra = {"key": "测试"}
    await client.execute("INSERT INTO test VALUES", [title, extra])
    rows = await client.fetch("SELECT * FROM test")
    assertEqual(rows[0][0], title)
    assertEqual(rows[0][1]["key"], extra["key"])



async def main():
    client = await get_client()
    await test_str(client)
    await client.close()


asyncio.run(main())
```

It runs without error under 2.5.0, while `assertEqual(rows[0][1]["key"], extra["key"])` will fail in 2.5.1

**Comment by Maksim-Burtsev** [comment]
PR #102 fix this issue

**Comment by maximdanilchenko** [comment]
Fixed in #102
